### PR TITLE
docs(readme.md): add repository badges

### PR DESCRIPTION
**Pull request by bl0cknumber**
- provides short information in badges
- clickable badge to opencollective

### CODE REVIEW

**Review by Shinigami92 — APPROVED** [review]
NICE

**Review by import-brain — APPROVED** [review]

**Review by prisis — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ Generate massive amounts of fake data in the Browser and Node.js.
 
 [![Chat on Discord](https://img.shields.io/discord/929487054990110771)](https://discord.com/invite/4qDjAmDj4P)
 [![Continuous Integration](https://github.com/faker-js/faker/actions/workflows/ci.yml/badge.svg)](https://github.com/faker-js/faker/actions/workflows/ci.yml)
+[![npm version](https://badgen.net/npm/v/@faker-js/faker)](https://www.npmjs.com/package/@faker-js/faker)
+[![npm downloads](https://badgen.net/npm/dm/@faker-js/faker)](https://www.npmjs.com/package/@faker-js/faker)
+[![contributors](https://img.shields.io/github/contributors/faker-js/faker)](https://github.com/faker-js/faker/graphs/contributors)
```
Comment (damienwebdev) [inline README.md:9]: Contributors is already a built-in feature of Github, I think we can remove this.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ Generate massive amounts of fake data in the Browser and Node.js.
...
+[![Open Collective](https://img.shields.io/opencollective/backers/fakerjs)](https://opencollective.com/fakerjs#section-contributors)
+[![sponsor](https://img.shields.io/opencollective/all/fakerjs?label=sponsors)](https://opencollective.com/fakerjs)
+[![Merged PRs](https://badgen.net/github/merged-prs/faker-js/faker)](https://github.com/faker-js/faker)
```
Comment (damienwebdev) [inline README.md:12]: I don't think a merged PR count is particularly useful.

File: README.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ Generate massive amounts of fake data in the Browser and Node.js.
...
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```
Comment (damienwebdev) [inline README.md:13]: License is already a built-in feature of Github, I think we can remove this.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

**Comment by Shinigami92** [comment]
@damienwebdev you are missing that this will also rendered on npm package site!
So I would use the contributors and license badge.
Also the PR badge shows a healthy activity, at least for now.
We could remove the PR counter when we reached 500+ PRs 🙂

**Comment by bl0cknumber** [comment]
> @damienwebdev you are missing that this will also rendered on npm package site! So I would use the contributors and license badge. Also the PR badge shows a healthy activity, at least for now. We could remove the PR counter when we reached 500+ PRs 🙂

Which one I should follow? your suggestion or @damienwebdev? or currently we stale it first?

**Comment by prisis** [comment]
For me only Merged PRs and contributors can be removed, license should be there for npm

**Comment by Shinigami92** [comment]
> For me only Merged PRs and contributors can be removed, license should be there for npm

Maybe we should just link it to the license file 
Many repos do it that way

**Comment by bl0cknumber** [comment]
Do we follow major consensus vote to remove merged PRs and contributors?

File: README.md [hunk 4]
```diff
@@ -6,7 +6,16 @@
   
   [![Chat on Discord](https://img.shields.io/discord/929487054990110771)](https://discord.com/invite/4qDjAmDj4P)
   [![Continuous Integration](https://github.com/faker-js/faker/actions/workflows/ci.yml/badge.svg)](https://github.com/faker-js/faker/actions/workflows/ci.yml)
-
+  [![Chat on Discord](https://img.shields.io/discord/929487054990110771)](https://discord.com/invite/4qDjAmDj4P)
+  [![Continuous Integration](https://github.com/faker-js/faker/actions/workflows/ci.yml/badge.svg)](https://github.com/faker-js/faker/actions/workflows/ci.yml)
```
Comment (damienwebdev) [inline README.md:10]: This is a duplicate.

File: README.md [hunk 5]
```diff
@@ -6,7 +6,16 @@
   
   [![Chat on Discord](https://img.shields.io/discord/929487054990110771)](https://discord.com/invite/4qDjAmDj4P)
   [![Continuous Integration](https://github.com/faker-js/faker/actions/workflows/ci.yml/badge.svg)](https://github.com/faker-js/faker/actions/workflows/ci.yml)
-
+  [![Chat on Discord](https://img.shields.io/discord/929487054990110771)](https://discord.com/invite/4qDjAmDj4P)
```
Comment (damienwebdev) [inline README.md:9]: This is a duplicate.

**Comment by damienwebdev** [comment]
> Do we follow major consensus vote to remove merged PRs and contributors?

We decided to move this change to v6.1.0, but I still want to outline a conversation that @Shinigami92 and I had in Discord:

1. We don't need the LICENSE badge, we have it built-in on both NPM and Github.
2. PR Merge count isn't a great ref to project stability, so it should be removed.
3. Contributors should be removed.

The rest seem good.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

**Comment by bl0cknumber** [comment]
> > Do we follow major consensus vote to remove merged PRs and contributors?
> 
> We decided to move this change to v6.1.0, but I still want to outline a conversation that @Shinigami92 and I had in Discord:
> 
> 1. We don't need the LICENSE badge, we have it built-in on both NPM and Github.
> 2. PR Merge count isn't a great ref to project stability, so it should be removed.
> 3. Contributors should be removed.
> 
> The rest seem good.

Cool, resolving, thanks @damienwebdev and @Shinigami92 😃

**Review by Shinigami92 — APPROVED** [review]

**Review by damienwebdev — APPROVED** [review]
Lgtm

**Comment by 1LISA2** [comment]
images
images (1)

**Comment by 1LISA2** [comment]
images

**Pull request merged**